02-peer-protocol.md

```diff
@@ -211,8 +212,8 @@ The sender:
 avoid double-spending of the funding transaction.
 
 The receiver:
-  - if the `chain_hash` value within the `open_channel` message is set to a hash
- of a chain unknown to the receiver:
+  - if the `chain_hash` value, within the `open_channel`, message is set to a hash
+ of a chain which is unknown to the receiver:
```

Comment:
This "which is" should be "that is" because the clause that follows is necessary to the sentence.

Reply:
OK, I'll fix this nit, rebase and merge now!